xbmc/platform/linux/AppParamParserWebOS.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "AppParamParserWebOS.h"
+
+CAppParamParserWebOS::CAppParamParserWebOS() : CAppParamParserLinux()
```

Comment:
Is there any value in inheritance from `CAppParamParserLinux`? It should be sufficient to override `CAppParamParser`.